client-side.md

```diff
@@ -22,16 +22,17 @@ See the separate [npm-packages.md](./npm-packages.md).
 
 -   Have an [.editorconfig](http://editorconfig.org/) file in your repo.
 -   Automate linting as part of your test phase (e.g. jshint, eslint,
-    csslint).
-    -   Use the [departmental linting rules](https://github.com/guardian/configs)
+    csslint) using the department linting rules:
+    - [@guardian/eslint-config](https://github.com/guardian/csnx/tree/main/libs/%40guardian/eslint-config)
+    - [@guardian/eslint-config-typescript](https://github.com/guardian/csnx/tree/main/libs/%40guardian/eslint-config-typescript)
+    - [@guardian/tsconfig](https://github.com/guardian/csnx/tree/main/libs/%40guardian/tsconfig)
+    - [@guardian/prettier](https://github.com/guardian/csnx/tree/main/libs/%40guardian/prettier)
 -   Don’t enforce coding styles in pull requests. Add linting dot files
     (e.g. .jshintrc) to your project instead.
 
 ## Dependencies
 
--   If starting a new project, favour ES6 modules unless you can justify
-    otherwise.
--   Prefer locking your dependencies (direct and transitive) using a lockfile rather than committing them into Git.
```

Comment:
I think we lost these two recommendations along the way? Are they so obvious now we don't need to spell them out, or would it be useful to capture them still?

Reply:
1. I both considered that at this point[ES6 (2015) is the standard](https://caniuse.com/es6), but also did not really fit below the **Dependencies** header. I would suggest a new **Modules** header if we want to capture more updated recommendations.
2. `pnpm`, `yarn` and `npm` now all create lockfiles by default. [We already mention lockfiles in `dependencies.md`](https://github.com/guardian/recommendations/blob/6dbdefa26502a72621279c53df51df44b28ecb99/dependencies.md?plain=1#L35)

Do you think this needs to be addressed in this PR?

Reply:
👍 Good points. No need to change anything in this PR.